lib/utils/utils.dart

```diff
@@ -12,7 +12,9 @@ import 'package:html_editor_enhanced/utils/shims/dart_ui.dart';
 void setState(
     bool mounted, void Function(Function()) setState, void Function() fn) {
   if (mounted) {
-    setState.call(fn);
+    WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
+      setState.call(fn);
+    });
```

Comment:
I am having issues with this change when the caret selection position is changed on web. The toolbar will not update the values, e.g. font size will not update when the text selection position is changed. Is there any particular reason why you added the `postFrameCallback`? I don't recall there being any issues with this `setState` function.

Reply:
that's because calling `setState` during `build` throws an exception
but now that we moved initialization logic in a future builder i am not sure if this throws or not

but font size not updating is not related to the changes i made, i think

Reply:
I didn't see any exceptions when this was removed. Also removing this caused the font size to update correctly, the set state was not firing due to the post frame callback. Do you mind testing on your end and making sure there are no side effects to removing this?